cypress/integration/login.js

```diff
@@ -1,92 +1,118 @@
-describe('Sourcebooks testLogIn', function() {
-    
-    var roles = [
-        { name: 'User', 'tabCount': 1 }, 
-        { name: 'Team Lead', tabCount: 2 }, 
-        { name: 'Manager', tabCount: 5 }, 
-        { name: 'Accountant', tabCount: 5}, 
-        { name: 'Admin', tabCount: 6 }
-    ];
+import LoginPage from '../obj/loginPage';
+import ClientsPage from '../obj/clients/clientsPage';
+import ClientCreatePage from '../obj/clients/clientCreatePage';
+import TasksPage from '../obj/tasks/tasksPage';
+import TaskCreatePage from '../obj/tasks/taskCreatePage';
+import TasksEditPage from '../obj/tasks/taskEditPage';
+import TaskEditPage from '../obj/tasks/taskEditPage';
+import TimeLoggingPage from '../obj/timeLogging/timeLoggingPage';
+import Common from '../obj/common';
 
-    function makeid(length) {
-        return 'Random_Name_' + Math.floor(Math.random() * 1000000000000);
-    }
+const loginPage = new LoginPage();
+const clientsPage = new ClientsPage();
+const clientCreatePage = new ClientCreatePage();
+const tasksPage = new TasksPage();
+const taskCreatePage = new TaskCreatePage();
+const taskEditPage = new TaskEditPage();
+const timeLoggingPage = new TimeLoggingPage();
+const common = new Common();
+    
+var roles = [
+    { name: 'User', 'tabCount': 1 }, 
+    { name: 'Team Lead', tabCount: 2 }, 
+    { name: 'Manager', tabCount: 5 }, 
+    { name: 'Accountant', tabCount: 5}, 
+    { name: 'Admin', tabCount: 6 }
+];
 
-    it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
-    })
+function makeId() {

```

Comment:
this function should not be in a test case, maybe some kind of utility class with all helping functions would be a good idea to have and keep them there